tutorials/34_Model_Based_Evaluation_of_RAG_Pipelines.ipynb

```diff
@@ -0,0 +1,4480 @@
+{
```

Comment:
<p>We should use the 2.0 Docs links here</p><ul><li><a href="https://docs.haystack.deepset.ai/v2.0/docs/enabling-gpu-acceleration" target="_blank">https://docs.haystack.deepset.ai/v2.0/docs/enabling-gpu-acceleration</a></li><li><a href="https://docs.haystack.deepset.ai/v2.0/docs/setting-the-log-level" target="_blank">https://docs.haystack.deepset.ai/v2.0/docs/setting-the-log-level</a></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
done